packages/eas-cli/src/graphql/queries/ChannelQuery.ts

```diff
@@ -0,0 +1,116 @@
+import gql from 'graphql-tag';
```

Comment:
We're trying to move away from the centralized `graphql` dir and put the graphQL in the same file if possible. Would it be possiblle to move these function inside as well?

Reply:
ah, gotcha, I thought this was the way to go. I'll move them back!